syntax = "proto3";

// Chat manager service
service ChatManager {
    // Chat with AI
    rpc ChatAI (RequestAIChat) returns (AIResponse);

    // Other queues
}

message RequestAIChat {
    string user_id = 1;
    string user_name = 2;
    string context = 3;
    string message = 4;
}

message AIResponse {
    string message = 1;
}